lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -79,8 +79,13 @@ def output_cop(cop_name, offense_count)
         output.puts
         cfg = self.class.config_to_allow_offenses[cop_name] || {}
 
-        output_cop_comments(output, cfg, cop_name, offense_count)
-        output_cop_config(output, cfg, cop_name)
+        # To avoid malformed YAML when potentially reading the config in
+        # #excludes, we use an output buffer and append it to the actual output
+        # only when it results in valid YAML.
+        buffer = StringIO.new
+        output_cop_comments(buffer, cfg, cop_name, offense_count)
+        output_cop_config(buffer, cfg, cop_name)
+        output.puts(buffer.string)
       end
 
       def output_cop_comments(output, cfg, cop_name, offense_count)
```

Comment:
Should the parameter (here, and in several methods below) still be called `output`, or should I rename them e.g. `buffer`?

Reply:
Or `output_buffer`? 😉

Reply:
Done

Reply:
I don't see any changes.

Reply:
Seems I forgot to push. Sorry.